app/layout/projects/show/map/index.tsx

```diff
@@ -131,18 +134,44 @@ export const ProjectMap: React.FC<ProjectMapProps> = () => {
     },
   });
 
-  const LAYERS = [PUGridLayer, PUCompareLayer, PlanningAreaLayer].filter((l) => !!l);
+  const FeaturePreviewLayers = useFeaturePreviewLayers({
+    features: selectedFeaturesData,
+    active: tab === 'features',
+    bbox,
+    options: {
+      selectedFeatures: selectedFeaturesIds,
+      ...layerSettings['features-preview'],
+    },
+  });
+
+  const selectedPreviewFeatures = useMemo(() => {
+    return selectedFeaturesData
+      ?.filter(({ id }) => selectedFeaturesIds.includes(id))
+      .map(({ featureClassName, id }) => ({ featureClassName, id }))
+      .sort((a, b) => {
+        const aIndex = selectedFeaturesIds.indexOf(a.id as string);
+        const bIndex = selectedFeaturesIds.indexOf(b.id as string);
+        return aIndex - bIndex;
+      });
+  }, [selectedFeaturesIds, selectedFeaturesData]);
+
+  const LAYERS = [PUGridLayer, PUCompareLayer, PlanningAreaLayer, ...FeaturePreviewLayers].filter(
+    (l) => !!l
+  );
 
   const LEGEND = useLegend({
     layers: [
+      ...(tab === 'features' && !!selectedFeaturesData?.length ? ['features-preview'] : []),
```

Comment:
I am not sure 100% if the user is able to see the feature preview layers even if they are in another tab, but to keep it simple, I would omit the tab condition and just go for _if there are features selected, we display them_ approach.